packages/core/admin/admin/src/translations/en.json

```diff
@@ -64,6 +64,9 @@
   "admin.pages.MarketPlacePage.plugin.info.text": "Learn more",
   "admin.pages.MarketPlacePage.plugin.info.label": "Learn more about {pluginName}",
   "admin.pages.MarketPlacePage.plugin.copy": "Copy install command",
+  "admin.pages.MarketPlacePage.plugin.copy.success": "Install command ready to pasted in your terminal",
```

Comment:
What about?

```suggestion
  "admin.pages.MarketPlacePage.plugin.copy.success": "Install command copied to your clipboard",
```

Reply:
We had something like that previously but there was feedback after the demo that the command should be more clear about what to do next. There is a thread on slack (let me know if you want the link). I do have a typo here though.

Reply:
Ok, great. Let's keep the discussion in Slack :)